src/consensus/pbft/libbyz/Client_proxy.h

```diff
@@ -99,24 +100,40 @@ class ClientProxy
   int rtimeout; // Timeout period in msecs
 
   // Maximum retransmission timeout in msecs
-  static const int Max_rtimeout = 200;
+  static const int Max_rtimeout = 3000;
```

Comment:
We should let these be configurable.

Reply:
I think these fall under this ticket: (https://github.com/microsoft/CCF/issues/562) these should be configured along with the other timeout values via CLI, same as raft config.